ngcore/ngcore.go

```diff
@@ -104,79 +105,49 @@ func (n *Neugram) GetOrNewSession(ctx context.Context, name string) *Session {
 	return s
 }
 
-func (s *Session) Exec(src []byte) error {
+func (s *Session) Exec(src []byte) ([][]reflect.Value, error) {
```

Comment:
Can this be collapsed into a []reflect.Value and still be displayed correctly? (That would avoid creating a slice in the common case of src being a single statement.)

Reply:
Ok.
Actually, as the jupyter backend is only interested in printing/evaluating the last statement, we could just return the last slice.

Reply:
done.